### PR TITLE
[Merged by Bors] - chore: remove some whitespace and two other things

### PR DESCRIPTION
**Pull request by adomani**
The fallout of an experiment (#14379) with linting whitespace.

To keep the review interesting, I invite you to find the two changes that are not whitespace removal.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Thanks, looks good to me!
maintainer merge

File: Mathlib/FieldTheory/RatFunc/Degree.lean [hunk 1]
```diff
@@ -53,13 +53,15 @@ theorem intDegree_one : intDegree (1 : RatFunc K) = 0 := by
...
-set_option linter.uppercaseLean3 false in #align ratfunc.int_degree_C RatFunc.intDegree_C
+set_option linter.uppercaseLean3 false in
+#align ratfunc.int_degree_C RatFunc.intDegree_C
 
 @[simp]
 theorem intDegree_X : intDegree (X : RatFunc K) = 1 := by
   rw [intDegree, num_X, Polynomial.natDegree_X, denom_X, Polynomial.natDegree_one,
     Int.ofNat_one, Int.ofNat_zero, sub_zero]
-set_option linter.uppercaseLean3 false in #align ratfunc.int_degree_X RatFunc.intDegree_X
+set_option linter.uppercaseLean3 false in
```
Comment (riccardobrasca) [inline Mathlib/FieldTheory/RatFunc/Degree.lean:63]: Are you sure this is the recommended style?

Comment (adomani) [reply Mathlib/FieldTheory/RatFunc/Degree.lean:63]: It is certainly the style that is used *every other time*!  :smile:

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**